NuSpec/Build/RedGate.AppHost.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup Condition="$(RedGateAppHostOutputPath) == ''">
+        <RedGateAppHostOutputPath>$(TargetDir)</RedGateAppHostOutputPath>
+    </PropertyGroup>
+    <Target Name="CopyRedGateAppHostFiles" AfterTargets="Build">
+        <ItemGroup>
+            <RedGateAppHostFiles Include="$(MSBuildThisFileDirectory)\..\tools\**\*.*" />
+        </ItemGroup>
+        <Copy SourceFiles="@(RedGateAppHostFiles)" DestinationFiles="@(RedGateAppHostFiles->'$(RedGateAppHostOutputPath)\%(RecursiveDir)%(Filename)%(Extension)')" SkipUnchangedFiles="true" />
```

Comment:
Do you mean something like:
```xml
<ItemGroup>
    <!--
    Copy the version specific DLLs to the output directory.
    -->
    <None Include="$(MSBuildThisFileDirectory)..\tools\**">
      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
  </ItemGroup>
```

This would include the items in the in the solution explorer. I'm not sure that is what you want from the exe files.